src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -224,7 +239,7 @@ public function getExtension(string $name)
     /**
      * Returns all registered extensions.
      *
-     * @return ExtensionInterface[] An array of ExtensionInterface
+     * @return array<string, ExtensionInterface> An array of ExtensionInterface
```

Comment:
please remove all extra descriptions on all modified lines:
~An array of ExtensionInterface~

Reply:
Done